nemo_skills/evaluation/metrics/mtbench_metrics.py

```diff
@@ -128,10 +115,10 @@ def get_metrics(self):
         return {self.agg_mode: metrics}
 
     def reset(self):
+        super().reset()
         self.scores = defaultdict(list)
-        self.total = 0
-        self.agg_mode = "greedy"
+        self.agg_mode = "pass@1"
 
-    def max_metrics_to_print(self):
+    def metrics_to_print(self):
         """We are only printing the averages, but all other metrics can still be found in metrics.json"""
-        return 4
+        return ['num_entries', 'avg_tokens', 'average', 'average_turn1', 'average_turn2']
```

Comment:
The math metric class return from this function a dict:
https://github.com/NVIDIA/NeMo-Skills/blob/d61d6f03929f907951f1cb61cf23c68d0bc0f25c/nemo_skills/evaluation/metrics/math_metrics.py#L147-L154

Confirming if this is correct.

Reply:
good catch, thanks!